main.go

```diff
@@ -125,6 +127,12 @@ func main() {
 		"Webhook server port (set to 0 to disable)",
 	)
 
+	flag.StringVar(&healthPort,
+		"health-port",
```

Comment:
Since this is similar to the metrics address, and that flag is called `metrics-addr`, let's call this `health-addr`

Reply:
done